test/CodeGenerator/InjectorGeneratorTest.php

```diff
@@ -58,4 +58,14 @@ public function testGeneratedInjectorIsValidCode()
         include $this->dir . '/GeneratedInjector.php';
         $this->assertTrue(class_exists($class, false));
     }
+
+    public function testSetCustomNamespace()
```

Comment:
In the over place I can see you've added `: void` return type. If so it should be also here.